civis/ml/_model.py

```diff
@@ -1219,6 +1220,13 @@ def predict(self, df=None, csv_path=None,
             RAM requested by the user for a single job.
         disk_space : float, optional
             disk space requested by the user for a single job.
+        dvs_to_predict : list of str, optional
+            If this is a multi-output model, you may list a subset of
+            dependent variables for which you wish to generate predictions.
+            This list must be a subset of the original `dependent_variable`
+            input. Ignoring some of the model's outputs will let predictions
```

Comment:
Instead of this sentence, what about something like "Scores for all outputs will always be calculated, but only the subset will be returned, allowing predictions to complete faster and use less disk space". I think it's good to make this explicit so that users know that asking for a subset won't affect their scores.

Reply:
It's true that we do currently let the model compute everything, then drop columns from the output array before writing to disk. But that seems like an implementation detail. If we were to somehow modify the models such that they only computed the requested labels, it would still look the same to the user. How about 

> The scores for the returned subset will be identical to the scores which those outputs would have had if all outputs were written, but ignoring some of the model's outputs will let predictions complete faster and use less disk space.

Reply:
Good point, that is more of an implementation detail. I like your revision.